device/mellanox/x86_64-mlnx_msn2700-r0/psu_sensors.json

```diff
@@ -0,0 +1,421 @@
+            "a1": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["1", "L"],
+                    "dps460-i2c-*-59": ["2", "R"]
+                }                
+            }
+        },
+        "MSN4600": {
+            "default": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["1", "L"],
+                    "dps460-i2c-*-59": ["2", "R"]
+                }
+            }
+        },
+        "MSN4600C": {
+            "default": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["2", "R"],
+                    "dps460-i2c-*-59": ["1", "L"]
+                }
+            },
+            "a1": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["1", "L"],
+                    "dps460-i2c-*-59": ["2", "R"]
+                }
+            },
+            "respin": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["2", "R"],
+                    "dps460-i2c-*-59": ["1", "L"]
+                }                
+            },
+            "respin_a1": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["1", "L"],
+                    "dps460-i2c-*-59": ["2", "R"]
+                }  
+            }
+        },
+        "MSN4700": {
+            "default": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["1", "L"],
+                    "dps460-i2c-*-59": ["2", "R"]
+                }
+            },
+            "a1": {
+                "bus": [
+                    "i2c-4",
+                    "i2c-1-mux (chan_id 3)"
+                ],
+                "chip": {
+                    "dps460-i2c-*-58": ["1", "L"],
+                    "dps460-i2c-*-59": ["2", "R"]
+                }                
+            }
+        },
+        "SN2201": {
+            "default": {
+                "bus": [
+                    "i2c-3",
+                    "i2c-1-mux (chan_id 1)"
+                ],
+                "chip": {
+                    "pmbus-i2c-3-58": ["1"],
+                    "pmbus-i2c-4-58": ["2"]
+                }
+            }
+        },
+        "SN4800": {
```

Comment:
is this a supported device? not sure we need it

Reply:
removed this entry as not supported